Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:20.04
+RUN apt-get update && apt-get upgrade --yes
+RUN apt-get install python3-pip python3.8 --yes
+RUN pip3 install --upgrade eth-ape
```

Comment:
Hmm.... I'd actually expect this to build from source. That way, you download the repo and can build the container for dev purposes

Reply:
so like this? 
```
RUN git clone https://github.com/ApeWorX/ape
RUN pip install -e ./ape
```

Reply:
I was thinking something like `python setup.py sdist` and then `pip install dist/eth-ape*` since this Dockerfile is in the same repo.

From CI/CD perspective, if we want to run a job to deploy the image to Dockerhub on release, there might be a race condition if relying on the build being available on pypi first

Reply:
Oh youd have to copy the source files to the container as well as use a `.dockerignore`

Reply:
Lol maybe `git` is better.

Reply:
I was testing this with git and i'm having a bunch of failures. I'll keep trying and post the failures if i can't *git* it to work

Reply:
as per conversation with @fubuloubu:
copy the files from the local directory into the docker image and then use pip install 
benefit: not relying on deployed version, it will be whatever is on the master branch and be easier to debug and avoid race condition

Reply:
I would agree with copying files from local directory as that is also easiest to build for the purposes of development and testing new features on a local instance via a built image.